persistence/docs/CHANGELOG.md

```diff
@@ -7,6 +7,100 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.0.9] - 2022-11-04
+
+- Changed the following exposed functions to lowercase non-exported functions
+- [./pocket/persistence/]
+  - GetActor
+  - GetActorFromRow
+  - GetChainsForActor
+  - SetActorStakeAmount
+  - GetActorStakeAmount
+  - GetCtxAndTx
+  - GetCtx
+  - SetValidatorStakedTokens
+  - GetValidatorStakedTokens
+- [./pocket/persistence/test/]
```

Comment:
I was hoping to do this in a separate PR, see bullet point 4:

![Screen Shot 2022-11-04 at 9 23 27 AM](https://user-images.githubusercontent.com/1892194/200025975-c128bd32-0878-40ae-8c17-d38c846402e9.png)

Again, just a note for future reference: separate asks -> separate PRs. Make PRs smaller if possible. It makes it easier for the reviewer, reduces cognitive load, makes bugs easier to track & reverse,etc...